src/ScnSocialAuth/Authentication/Adapter/HybridAuth.php

```diff
@@ -319,6 +319,12 @@ protected function instantiateLocalUser()
     protected function facebookToLocalUser($userProfile)
     {
         if (!isset($userProfile->emailVerified)) {
+            //some facebook user's dont have email address
+            if ($userProfile->email == ''){
+              $userProfile->email = $userProfile->identifier.'@facebook.com';
+              $userProfile->email = $userProfile->emailVerified;
```

Comment:
Ya sorry about the typo there ... it should be $userProfile->emailVerified = $userProfile->email. As mentioned earlier some users are only registered with their mobile numbers in facebook and I wanted to allow those users to register into my app as well.